packages/@aws-cdk/app-delivery/package.json

```diff
@@ -46,6 +51,21 @@
     "fast-check": "^1.7.0",
     "pkglint": "^0.17.0"
   },
+  "repository": {
+  },
+  "devDependencies": {
+    "@aws-cdk/aws-codepipeline": "^0.17.0",
+    "@aws-cdk/aws-s3": "^0.17.0",
+    "cdk-build-tools": "^0.17.0",
+    "cdk-integ-tools": "^0.17.0",
```

Comment:
Some dependencies are duplicated here (`cdk-build-tools`, `cdk-integ-tools`, `@aws-cdk/aws-s3`, `@aws-cdk/aws-codepipeline`) - I would remove the extra ones  so that the `@aws-cdk` namespace ones are grouped at the beginning.